recipes/AstroGlue/yum_requirements.txt

```diff
@@ -0,0 +1,3 @@
+xxorg-x11-server-Xvfb
+gtk2-devel
```

Comment:
Why is this needed? We have this as a conda package.

Reply:
This should be manageable without the `gtk2-devel` line.

Reply:
Thank you! 
I will remove `gtk2-devel` line, re-run the pipelines and add a mention for review once that is done.

Reply:
In the absence of the `gkt2-devel` line in the `yum_requirements.txt`, pipelines testing for Windows environment fails with the following error:
![image](https://github.com/user-attachments/assets/948f1fec-bef7-4745-a866-d5e527de6b7a)

All the additional requirements seem to be necessary for a successful run. I added these requirements upon reviewing the glue-qt feedstock (which is also used in AstroGlue) - https://github.com/conda-forge/glue-qt-feedstock/blob/main/recipe/yum_requirements.txt

Please do let me know if there is any other way to resolve this.
Thanks in advance!

Reply:
@xhochy 
Hi, I just wanted to check if there is anything else that I can do to get this resolved and re-request a review. The requirements files still includes `gkt2-devel` line as the Windows environment testing fails without it. 

Thank you in advance!

Reply:
`yum_requirements.txt` shouldn't have influence on Windows tests.

Reply:
Hello!
I removed the `gkt2-devel` line from the `yum_requirements` file. All the pipelines tests have run successfully now. I apologise for the confusion earlier. Please do let me know if there's anything left to be done by me. Thank you so much!